semanticscholar/ApiRequester.py

```diff
@@ -40,7 +37,7 @@ def timeout(self, timeout: int) -> None:
         retry=retry_if_exception_type(ConnectionRefusedError),
         stop=stop_after_attempt(10)
     )
-    def get_data(
```

Comment:
Still considering the need to minimize the impacts of breaking changes, I think it's important to keep the get_data method sync version, since it's a public method and someone might be using it. Therefore, the ideal would be to have a sync version, using the same approach of calling the async version, and deprecate the sync version to be removed in the future. (I know the intention is not for the method to be used directly, but you never know how it's actually being used).

```python
warnings.warn(
  "get_data() is deprecated and will be disabled in the future," +
  " use the async version instead.",
  DeprecationWarning)
```